scripts/apps/master-desk/components/UsersComponent.tsx

```diff
@@ -61,7 +63,8 @@ class UsersComponent extends React.Component<IProps, IState> {
 
             this.setState({
                 roles: roles._items,
-                usersByRole: users._items,
+                usersByRole: users._items.filter((item) => item.role != null),
```

Comment:
This is a good use case for lodash's `partition` method.

User count will be < 100 in most cases so iterating the same thing twice doesn't hurt in terms of performance, but it's a good habit to do it more efficiently when it doesn't require significantly more effort.

---

Another thing - it'd better to call it `usersWithRole`(especially when you have a opposite value called `usersWithoutRole`). When naming convention of `{entity}By{Field}` is used, I expect an object/map. In case of `usersByRole`, it'd expect `{[roleId: string]: IUser}`